### DISCUSSION TITLE
about automatic remove console.log

### DISCUSSION THREAD

**Discussion by 19Qingfeng** [remix-run/remix#6521] [Q&A] [answered]
I have many non-standard third-party node_modules packages that have a lot of console.log. I want to remove these console.log prints during production builds, but I am not sure how Remix considers this feature. I have not found this option in Remixconfig.js yet

**Comment by kael** [comment, marked as answer]
A quick and dirty solution: put the following code under the last `import` of the _`root.jsx`_:

```js
(() => { if (process.env.NODE_ENV === 'production') console.log = () => {}; })();
```

**Comment by 19Qingfeng** [comment]
I added --clear to choose whether or not to clear when building #6536

**Reply by 19Qingfeng** [reply]
> I added --clear to choose whether or not to clear when building #6536

@MichaelDeBoey What do you think of this feature
